src/exampleTypes/ResultTable/index.js

```diff
@@ -89,7 +89,7 @@ let ResultTable = ({
   }
 
   return (
-    <>
+    <div data-path={node.path}>
```

Comment:
Tables are a bit delicate. I don't know if wrapping this one on a div will wreak havoc somewhere. You can pass `data-path` to the `Table` component instead.

Reply:
@stellarhoof I moved it to the `Table`